StardewCropCalc/com.elizabethburgessamt.stardewcropcalc/seeds/Seed.java

```diff
@@ -2,4 +2,22 @@
 
 public class Seed {
 	//TODO: Create proper data object
+	private string crop;
+	private int growth;
+	private int regrowth;
+	private int cost;
+	private int sell;
+	private boolean trellis;
+	
+	public Seed(string crop, int growth, int regrowth, int cost, int sell, string trellis){
+		this.crop = crop;
+		this.growth = growth;
+		this.regrowth = regrowth;
+		this.cost = cost;
+		this.sell = sell;
+		if (trellis == "TRUE"){

```

Comment:
I'd rather see this resolved in the SeedMaker class to a boolean instead of passing it like a String.  Another hint is that when you have an if-else and you'd rather shortcut it, you can do something like the following:
this.trellis = false;
if ("TRUE".equals(trellis)) { //also, this is the way to check for equivalence of Strings.
    this.trellis = true;
}